src/appmixer/hubspot/plugin.js

```diff
@@ -1,13 +1,34 @@
 'use strict';
+const Hubspot = require('./Hubspot');
 
 module.exports = async context => {
 
     const config = require('./config')(context);
+    const { appId, apiKey } = config;
+    const { clientId, clientSecret } = context.config;
 
-    if (!(config.appId && config.apiKey)) {
-        context.log('error', 'Hubspot module not configured properly, missing appId or apiKey.');
+    /** Is this AuthHub or Engine pod? */
+    const isAuthHubPod = !!process.env.AUTH_HUB_URL && !process.env.AUTH_HUB_TOKEN;
+
+    // When using AuthHub, the configuration is not needed in the Engine.
+    // How do we know if we are using AuthHub? We have `clientId` but not `clientSecret` and no `appId` or `apiKey`.
+    const isAuthHubInUse = !appId && !apiKey && !!clientId && !clientSecret;
```

Comment:
@martin-krcmar please review the logic that determines which pod we are in and which config (AuthHub or backoffice) is used.